internal/server/workflow_vars.go

```diff
@@ -51,12 +52,18 @@ func NewWorkflowVars(client WorkflowVarsClient, r *http.Request, envVars Environ
 		return nil, err
 	}
 
+	paProTeamSelection := createReassignDeputyDropDownList(teamSelection, []string{"PA", "PRO"}, selectedTeam)
```

Comment:
This feels like something that should be done in the deputies handler as it is specific to the deputies page. We should avoid throwing everything into the WorkflowVars, otherwise it will just be a catch all and we'll lose any separation of concerns.